cmd/alerts/silence/add_silence.go

```diff
@@ -117,7 +119,7 @@ func fetchAllAlerts(kubeconfig *rest.Config, clientset *kubernetes.Clientset) []
 	return fetchedAlerts
 }
 
-func AddAlertNameSilence(alertID []string, duration, comment, username string, kubeconfig *rest.Config, clientset *kubernetes.Clientset) {
+func AddAlertNameSilence(alertID []string, duration, comment, username string, kubeconfig *rest.Config, clientset *kubernetes.Clientset) error {
```

Comment:
Line 72 in this file has a call to this function without handling the error - let's wrap that